index.html

```diff
@@ -0,0 +1,469 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Youtube Page</title>
+  </head>
+
+  <body>
+    <nav class="flex">
+      <div id="left-navbar" class="flex">
+        <a href="#"><img src="assets/hamburger.png" alt="" /></a>
+        <a href="#"><img src="assets/youtubelogos.png" alt="" /></a>
+      </div>
+      <div id="mid-navbar" class="flex">
+        <input id="search-bar" type="text" placeholder="Search" />
+        <button id="search-button">
+          <i class="fa fa-search" aria-hidden="true"></i>
+        </button>
+      </div>
+
+      <div id="right-navbar" class="flex">
+        <a href="#"><img src="assets/nav1.png" alt="" /></a>
+        <a href="#"><img src="assets/nav2.png" alt="" /></a>
+        <a href="#"><img src="assets/nav4.png" alt="" /></a>
+        <a href="#"><img src="assets/nav3.png" alt="" /></a>
+      </div>
+    </nav>
+
```

Comment:
You need to wrap the main part of your webpage inside the main tag. You can check the usage of the semantic tag [here](https://www.w3schools.com/html/html5_semantic_elements.asp)